### ISSUE TITLE
[Alan] Scene 5 Audio - Order change

### ISSUE THREAD

**Issue by bakkenb** [w3c/wai-people-use-web-videos#153]
[ED]
I think the message is being conveyed backward. See what you think about this order instead.

Current:

> Unfortunately, many websites and apps don’t work well with keyboard. Often the focus indicator, which highlights the currently active object on the screen, is not visible or it jumps around completely out of order. Usually, the Tab-key should take me object by object. ...

Suggestion:

> Unfortunately, many websites and apps don’t work well with keyboard. The Tab-key should take me through each page object by object in a logical order. But to often the focus indicator, which highlights the currently active object on the screen, is not visible or jumps around completely out of order. ...

Feel free to use or leave as is. For me the current version just seamed a little backward. Feel free to tweak if it is useful.

**Comment by MicheleAWilliams-A11y** [comment]
+1 here - noticed the same and think we should change it

**Comment by nitedog** [comment]
Changed as suggested.
